homeassistant/components/harmony/switch.py

```diff
@@ -65,10 +56,28 @@ def is_on(self):
 
     async def async_turn_on(self, **kwargs: Any) -> None:
         """Start this activity."""
+        async_create_issue(
+            self.hass,
+            DOMAIN,
+            "deprecated_switches",
+            breaks_in_ha_version="2024.6.0",
+            is_fixable=False,
```

Comment:
Repair issues created for deprecated services should normally be fixable as the user can fix them by removing the usage from automations and scripts. Search for other repair issues with `is_fixable=True`.